### PR TITLE
Fixed the css for the upload-buttons, as well as a filter/search-function for teacher

### PR DESCRIPTION
**Pull request by normansebastian**
det verkar inte som att det går att ladda upp filer till "aktiviteter", det gick förut

### CODE REVIEW

File: MOAS-LMS/Views/Account/Teacher.cshtml [hunk 1]
```diff
@@ -4,38 +4,58 @@
     ViewBag.Title = "Teacher";
 }
 
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (tollyx) [inline MOAS-LMS/Views/Account/Teacher.cshtml]: Jquery inkluderas redan på sidan, kolla i botten av `_Layout.cshtml`

File: MOAS-LMS/Views/Account/Teacher.cshtml [hunk 2]
```diff
@@ -4,38 +4,58 @@
...
+<link href="~/Content/Site.css" rel="stylesheet" />
```
Comment (tollyx) [inline MOAS-LMS/Views/Account/Teacher.cshtml]: Samma här, inkluderas redan i toppen av `_Layout.cshtml`

File: MOAS-LMS/Views/Account/Teacher.cshtml [hunk 3]
```diff
@@ -4,38 +4,58 @@
...
+<script>
+    $(document).ready(function () {
+        $("#myInput").on("keyup", function () {
+            var value = $(this).val().toLowerCase();
+            $("#myTable tr").filter(function () {
+                $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);
+            });
+        });
+    });
+</script>
```
Comment (tollyx) [inline MOAS-LMS/Views/Account/Teacher.cshtml]: Du borde skapa en egen `.js` fil för det här och inkludera den istället för att skriva js direkt i en html tagg.

Om du inkluderar den i `section scripts {}` så behöver du inte inkludera en egen jquery heller, för då inkluderas jquery innan ditt script. (vilket var troligen orsaken varför du behövde inkludera jquery själv i det här fallet)

File: MOAS-LMS/Views/Course/Details.cshtml [hunk 4]
```diff
@@ -5,25 +5,31 @@
     var canEdit = User.IsInRole("Admin");
 }
 
+<link href="~/Content/Site.css" rel="stylesheet" />
```
Comment (tollyx) [inline MOAS-LMS/Views/Course/Details.cshtml]: Inkluderas redan i toppen av `_Layout.cshtml`

File: MOAS-LMS/Views/Course/_Activity.cshtml [hunk 5]
```diff
@@ -32,9 +32,15 @@
...
+                <p></p>
                 <form action="/Document/UploadToActivity/@Model.Id" method="post" enctype="multipart/form-data">
+                    @*<input type="file" name="file" value="" />
+                    <input type="submit" value="Upload" />*@
                     <input type="file" name="file" value="" />
-                    <input type="submit" value="Upload" />
+                    <div class="fileUpload btn btn-primary" >
+                        <span>Upload</span>
+                        <input type="submit" class="upload" />
+                    </div>
```
Comment (tollyx) [inline MOAS-LMS/Views/Course/_Activity.cshtml]: Du glömde upload-knappen för att lämna in assignments, precis under de här raderna.

Det skulle kanske också se bättre ut om du gav `<input>` `display: inline-block`?

File: MOAS-LMS/Views/Course/_Module.cshtml [hunk 6]
```diff
@@ -36,9 +36,15 @@
                     @Html.ActionLink("Delete", "Delete", "Module", new { id = Model.Id }, null) |
                     @Html.ActionLink("Create a new Activity", "Create", "Activity", new { id = Model.Id }, null)
                 </span>
+                <p></p>
```
Comment (tollyx) [inline MOAS-LMS/Views/Course/_Module.cshtml]: ???

File: MOAS-LMS/Content/Site.css [hunk 7]
```diff
@@ -66,3 +66,37 @@ textarea {
...
+
+.fileUpload {
+    position: relative; right: 10px;
+    overflow: hidden;
+    margin: 10px;
+    display: inline-block;
+}
+
+.fileUpload input.upload {
+    position: absolute;
```
Comment (tollyx) [inline MOAS-LMS/Content/Site.css]: Jag rekommenderar att undvika `position: absolute` om du inte har någon specifik orsak för att använda det.

Comment (tollyx) [reply MOAS-LMS/Content/Site.css]: Är det pga någo konstig bootstrap grejs?

Kan du inte sätta bootstrap-knapp-klasserna direkt på `<input>` taggen?

File: MOAS-LMS/Views/Account/Teacher.cshtml (hunk 3 above)
Comment (tollyx) [reply MOAS-LMS/Views/Account/Teacher.cshtml]: Du har redan skapat en egen `.js` fil men du använder den inte?

File: MOAS-LMS/Views/Course/_Module.cshtml (hunk 6 above)
Comment (normansebastian) [reply MOAS-LMS/Views/Course/_Module.cshtml]: det är för att "Edit | Delete | Create a new Activity" ska ha samma avstånd till "choose file"-knappen som "choose file"-knappen för Course (och Edit | Return to Course Overview, osv), och samma för Module. Utan <p>-taggarna är avståndet smalare.
avstand

File: MOAS-LMS/Views/Account/Teacher.cshtml (hunk 3 above)
Comment (normansebastian) [reply MOAS-LMS/Views/Account/Teacher.cshtml]: done

File: MOAS-LMS/Content/Site.css (hunk 7 above)
Comment (normansebastian) [reply MOAS-LMS/Content/Site.css]: position: relative
position_relative

File: MOAS-LMS/Views/Course/Details.cshtml (hunk 4 above)
Comment (normansebastian) [reply MOAS-LMS/Views/Course/Details.cshtml]: borta

File: MOAS-LMS/Content/Site.css (hunk 7 above)
Comment (normansebastian) [reply MOAS-LMS/Content/Site.css]: done

File: MOAS-LMS/Views/Course/_Activity.cshtml (hunk 5 above)
Comment (normansebastian) [reply MOAS-LMS/Views/Course/_Activity.cshtml]: jag försökte få "upload"-knappen att ligga bredvid "choose file" tidigare, men det krockade iaf med texten av den 'valda' filen, så jag valde att flytta "upload"-knappen med css.
choose_file

File: MOAS-LMS/Views/Course/_Module.cshtml (hunk 6 above)
Comment (tollyx) [reply MOAS-LMS/Views/Course/_Module.cshtml]: Det där borde egentligen lösas med css.

Lägg till antingen `padding-top` eller `margin-top` på form-elementet
